.packit.yaml

```diff
@@ -130,30 +131,38 @@ jobs:
   # run all tests for all python versions on all fedoras
   - job: tests
     trigger: pull_request
-    identifier: upstream-all-fedora
-    tmt_plan: /upstream/all
+    identifier: distro-fedora-all
+    tmt_plan: /distro/all_python
     packages: [python-podman-fedora]
     targets:
       - fedora-all
+
+  # run tests for the rawhide python version using podman-next packages
+  - job: tests
+    trigger: pull_request
```

Comment:
I guess this should be changed to `ignore` unless we're ok with them failing often.

Reply:
Alternatively, if you use this: https://packit.dev/posts/manual-triggering#triggering-test-jobs-based-on-labels-and-identifiers , you could run the default set of tests in all cases and run the podman-next test only when the pomdman-next / pnext label is added to the PR. That can be a manual action. When the label is not present, that test will simply not run and will show a rather misleading `waiting for RPM build to succeed` status and just not run. Maybe better than a failed test though.

Reply:
that is perfect!

Reply:
they should be passing now, though, so I'll fix them first